docs/decision-making-process.md

```diff
@@ -0,0 +1,24 @@
+
+**Voting/Recommendation Platform:** Voting will take place via GitHub issue, where each Committee member will be invited to comment and indicate their preference using the following options: "Agree", "Disagree", "Abstain", or request further clarification. This will often occur through comments and [using “emoji” reactions on a GitHub issue](https://github.blog/news-insights/product-news/add-reactions-to-pull-requests-issues-and-comments/). Recommendations on any matter discussed with the Committee will also be made via GitHub issues.
+
+**Modalities:** To encourage active participation and input within the Committee, two modes will be available for decision-making and recommendation: 1) **Synchronously:** During committee meetings, where notes can be live-recorded on the issue, and 2) **Asynchronously:** Through engagement on the GitHub issue.
+Anonymity is not supported in either mode. For anonymous feedback, the Committee can contact the Executive Leadership team via email or Slack, addressing either the entire team or a specific member.
+
+**Types of Voting:** Active members can express their position on a GitHub issue by commenting with "Agree," "Disagree," or "Abstain" regarding the presented initiative. Members who "Disagree" or "Abstain" must provide a written justification (in a comment) to help improve the resources, initiative, or other discussion points. This enables the Committee to reach collective decisions or formulate recommendations for the executive team to consider.
+
+**Informed Voting:** The Committee will discuss and address each agenda item, providing an opportunity for all perspectives to be heard and expressed. The proposed modalities will allow for consideration of both supporting and opposing arguments before concluding the decision-making and recommendation process within the designated timeframe.
+
+**Committee’s Decision-Making:** Decisions will require approval by at least a two-thirds affirmative vote of active Committee members. It is crucial to understand that the Committee's decision-making process allows them to make recommendations, while the OLS Board of Directors retains final responsibility for actions affecting the organisation's activities.
+
+**Tied Votes:** In the event of tied alternatives, the Board of Directors will be responsible for resolving the issue by casting a vote between the tied options.
```

Comment:
```
**Tied Votes:** In the event of tied alternatives, the Board of Directors will be responsible for resolving the issue by casting a vote between the tied options.
```

Not quite sure who you mean by board of directors here - is this the OLS leadership team or the governance advisory committee? let's rename or link to a definitions doc

Reply:
How about this?

```suggestion
**Tied Votes:** In the event of an indecisive voting outcome from the Governance Advisory Committee's voting process, the Board of Directors will be responsible for resolving the issue by casting a vote between the tied options.
```

Reply:
much clearer!